repos/system_upgrade/common/libraries/dnfplugin.py

```diff
@@ -334,8 +334,9 @@ def install_initramdisk_requirements(packages, target_userspace_info, used_repos
     """
     Performs the installation of packages into the initram disk
     """
-    with _prepare_transaction(used_repos=used_repos,
-                              target_userspace_info=target_userspace_info) as (context, target_repoids, _unused):
+    mount_binds = ['/:/installroot']
```

Comment:
yes. it's needed.

Reply:
Yeah I know that, but I was asking if there are any other considerations e.g. is it possible that some other bind mounts are needed in some special cases?

Reply:
this should be the only one needed bind mount. and it is needed only in case of use of local repositories, but I do not see it harmful to bind mount it unconditionally